web/mailman-web/settings.py

```diff
@@ -281,8 +281,8 @@
 #
 COMPRESS_PRECOMPILERS = (
    ('text/less', 'lessc {infile} {outfile}'),
-   ('text/x-scss', 'sass -t compressed {infile} {outfile}'),
-   ('text/x-sass', 'sass -t compressed {infile} {outfile}'),
+   ('text/x-scss', 'sassc -t compressed {infile} {outfile}'),
+   ('text/x-sass', 'sassc -t compressed {infile} {outfile}'),
```

Comment:
No. libsass is meant to be an inplace replacement for ruby-sass, but for some reason the executable seems to differ.